code/game/objects/items/weapons/defib.dm

```diff
@@ -428,4 +428,119 @@
 			update_icon()
 		else
 			user << "<span class='notice'>You need to target your patient's chest with [src].</span>"
+			return
+
+/obj/item/weapon/borg_defib
+	name = "defibrillator paddles"
+	desc = "A pair of mounted paddles with flat metal surfaces that are used to deliver powerful electric shocks."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "defibpaddles"
+	item_state = "defibpaddles"
+	force = 0
+	w_class = 4
+	flags = NODROP
+	var/revivecost = 1000
+	var/cooldown = 0
+	var/busy = 0
+
+/obj/item/weapon/borg_defib/attack(mob/M, mob/user)
```

Comment:
And see https://github.com/tgstation/-tg-station/pull/7734#issuecomment-74396253 for ideas of a better way to do it.

But ask @paprka, because the defibrillator seems to be his thing.
Ask him to make some global procs that can be used instead.